README.md

```diff
@@ -1,6 +1,8 @@
 # workos-rb
 WorkOS official Ruby gem for interacting with WorkOS APIs
 
+[![codecov](https://codecov.io/gh/workos-inc/workos-ruby/branch/master/graph/badge.svg)](https://codecov.io/gh/workos-inc/workos-ruby)
```

Comment:
How about putting the badget next to the main header?

https://github.com/workos-inc/workos/raw/master/README.md